accessibilityTest/AccessibilityErrorReport.ts

```diff
@@ -1,24 +1,44 @@
 import * as axe from 'axe-core';
 import { $$, Component, ErrorReport } from 'coveo-search-ui';
-import { addQueryFilter, afterQueryError, getResultsColumn, getRoot } from './Testing';
+import { addQueryFilter, afterQueryError, getResultsColumn, getRoot, afterQuerySuccess } from './Testing';
 
 export const AccessibilityErrorReport = () => {
   describe('ErrorReport', () => {
-    beforeEach(() => {
-      // !BOOM
-      addQueryFilter('$qre()');
-    });
-
-    it('should be accessible', async done => {
+    const buildErrortReportElement = () => {
       const errorReportElement = $$('div', {
         className: Component.computeCssClassName(ErrorReport)
       });
 
       getResultsColumn().appendChild(errorReportElement.el);
-      await afterQueryError();
-      const axeResults = await axe.run(getRoot());
-      expect(axeResults).toBeAccessible();
-      done();
+    };
+
+    describe('with a query that triggers an error', () => {
```

Comment:
Does this need to be in a `describe` block if it only contains one test? or could it be consolidated in an `it` block?

Same comment for line 30.